com.alizadicomviewer.AlizaMS.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```

Comment:
There is a desktop file upstream

https://github.com/AlizaMedicalImaging/AlizaMS/tree/master/package/archive/usr/share/applications

Why adding this?

Reply:
Such desktop entry doesn't contain the icon name in the reverse-DNS style.
In addition the Exec option points to the executable and not to the script.

Reply:
Could I use desktop-file-edit?

Reply:
as well as `rename-desktop-file`, yes.